apps/cmp.c

```diff
@@ -1109,7 +1109,7 @@ static OSSL_CMP_SRV_CTX *setup_srv_ctx(ENGINE *engine)
         goto err;
 
     if (opt_send_error)
-        (void)ossl_cmp_mock_srv_set_send_error(srv_ctx, 1);
+        (void)ossl_cmp_mock_srv_set_sendError(srv_ctx, 1);
```

Comment:
This name change seems a bit odd.  We don't usually go for camel casing.
But still, that's the internal name...

Reply:
I know we generally do not use camelCase, except where a spec being implemented (here: RFC 4210 etc.) uses it.
As I wrote above, I changed it here just for consistency with the related functions.

> But still, that's the internal name...

Yes.